docs/contributing.rst

```diff
@@ -0,0 +1,7 @@
+Contributing to Ocean
+================================
+
+D-Wave welcomes contributions to Ocean projects.
+
+See how to contribute at
+`Ocean Contributors <http://dw-docs.readthedocs.io/en/latest/glossary.html>`_\ .
```

Comment:
I think this link should point somewhere else

Reply:
Yes, it's a placeholder until there's some content for that topic

Reply:
Because I want to start using it, I think it might be safe to change the placeholder to http://dw-docs.readthedocs.io/en/latest/contributing.html